src/system.cpp

```diff
@@ -106,6 +106,35 @@ bool System::command_response(const char * value, const int8_t id, JsonObject &
     return true;
 }
 
+// output all the EMS devices and their values, plus the sensors and any custom things
+// basically dump everything we know about
```

Comment:
Nice, but the other `device/values` commands uses the shortnames (`info` the long names). `system` is always a bit special, so it's ok.

Reply:
hmm, that's a good point. So how about calling the system command 'allinfo' instead of 'values' ?

Or stick to `values` and show the shortname, which is less readable, and not translated, but still consistent.

Reply:
made it allvalues